tests/test_entry_point.py

```diff
@@ -5,21 +5,22 @@
     create_compute_conditional_continuation_value,
     get_lcm_function,
 )
-from lcm.example_models.basic_example_models import (
-    PHELPS_DEATON,
-    PHELPS_DEATON_FULLY_DISCRETE,
-    PHELPS_DEATON_WITH_FILTERS,
-    phelps_deaton_utility,
-)
 from lcm.model_functions import get_utility_and_feasibility_function
 from lcm.process_model import process_model
 from lcm.state_space import create_state_choice_space
 from pybaum import tree_equal, tree_map
 
+from tests.test_models.deterministic import (
+    BASE_MODEL,
+    BASE_MODEL_FULLY_DISCRETE,
+    BASE_MODEL_WITH_FILTERS,
+    utility,
+)
+
 MODELS = {
```

Comment:
Unless we'll use this more than once, I find this more confusing than helpful.

Reply:
Do you mean the `models` dictionary?